### PR TITLE
Add phpmd to travis

### PR DESCRIPTION
**Pull request by stefk**
This PR adds a minimal phpmd check -- only unused private methods for now -- to the travis build. The goal is to progressively add rules to:

- eliminate dead code
- eliminate problematic functions like `exit`, `eval`, `var_dump`, etc.
- limit code complexity (class/method length, dependencies)

cc @claroline/commiters

### CODE REVIEW

**Comment by LaurentGruber** [comment]
LGTM

**Comment by stefk** [comment]
I've removed unused private methods from the code base as a starting point. This needs a careful review. @kitan1982, could you check that the scorm listener is ok?

File: plugin/collecticiel/Controller/DropController.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
  * Date: 22/08/13
  * Time: 09:30.
  */
+
 namespace Innova\CollecticielBundle\Controller;
```
Comment (LaurentGruber) [inline plugin/collecticiel/Controller/DropController.php]: @claroline/innova could you also check that there isn't problem with the collecticiel ?

**Comment by kitan1982** [comment]
LGTM

I had copied these private methods to a library class and forgotten to delete the original ones.

**Comment by ngodfraind** [comment]
This is ok for the CoreBundle.
LGTM.

**Comment by LaurentGruber** [comment]
LGTM

**Pull request merged**